src/docs/asciidoc/core/core-beans.adoc

```diff
@@ -6642,7 +6642,7 @@ annotated classes. The following example shows how to do so:
 [source,java,indent=0,subs="verbatim,quotes",role="primary"]
 .Java
 ----
-	@Component
+	@Configuration
```

Comment:
The documentation is correct.
We can define methods with **@Bean** annotation inside any **@Component** classes.